exchanges/okgroup/okgroup.go

```diff
@@ -366,9 +368,22 @@ func (o *OKGroup) GetSpotFilledOrdersInformation(request GetSpotFilledOrdersInfo
 }
 
 // GetSpotMarketData Get the charts of the trading pairs. Charts are returned in grouped buckets based on requested granularity.
-func (o *OKGroup) GetSpotMarketData(request GetSpotMarketDataRequest) (resp GetSpotMarketDataResponse, _ error) {
+func (o *OKGroup) GetSpotMarketData(request *GetSpotMarketDataRequest) (resp GetSpotMarketDataResponse, err error) {
```

Comment:
Maybe generalise the name because we are obtaining multiple asset types, including a change in the param request struct.

Reply:
Yeah there is GetSwapMarketData() & GetFuturesMarketData() i might remove them two and unify it as the only difference is a single path to submit a request